packages/mobile-top-nav/src/css/media-menu.js

```diff
@@ -0,0 +1,69 @@
+import { css } from 'lit-element';
+
+export default () => (
```

Comment:
I mentioned this in the other MR, but these external CSS files are a big red flag for web components. This should be inside the web component that it is styling.

Reply:
This is an organization style preference, not a way of circumventing the styling method inherent in WCs. The CSS will rarely be worked on after initially styling the component, so to me it's nice to move it out of the way of scanning the component's functionality. This also sets up the ability to share styles with components, as outlined [here](https://lit-element.polymer-project.org/guide/styles#sharing-styles)

Reply:
IMO this creates a huge disconnect between the markup it's supposed to be styling. It also creates a bunch of unnecessary additional files separated from their intended use. 

You can't assume that CSS will rarely be worked on after completion. It happens all the time and the person that works on that CSS isn't going to want to jump back and forth between files to do their updates. I certainly don't.

This doesn't set us up for sharing styles and I contend that we shouldn't be doing that at all. You're just taking a specific component's style and putting it in an external file.

Sharing CSS styles in web components is piercing the web component veil and is going to lead to a "shared" style sheet that no one wants to touch.